pkgs/development/python-modules/manuf/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, runCommand
+, python3
+}:
+
+buildPythonPackage rec {
+  pname = "manuf";
+  version = "1.1.5";
+
+  src = fetchFromGitHub {
+    owner = "coolbho3k";
+    repo = "manuf";
+    rev = "${version}";
+    sha256 = "sha256-3CFs3aqwE8rZPwU1QBqAGxNHT5jg7ymG12yBD56gTNI=";
+  };
+
+  # Tests appeared to be broken
+  doCheck = false;
```

Comment:
Why doesn't the test work?
Is it because of the sandbox environment?

Reply:
While executing test via nix build I get
```
Traceback (most recent call last):
  File "/build/source/manuf/manuf.py", line 278, in _get_mac_int
    return int(mac_str, 16) << self._bits_left(mac_str)
ValueError: invalid literal for int() with base 16: 'test'
```

Which is most likely due to an incorrect way of launching tests, and can be fixed.

But even if it is fixed, there are still test that fail in general https://github.com/coolbho3k/manuf/actions. Update functionality seem to be broken, and is not required due to the nix way of updating packages. Of course, it is possible to patch tests, removing the broken one. But I thought nix test will be sufficient in that case, since this package is quite simple. 

I have extended comment why the tests has been turned off, but you think it is important to get them working, I can try doing that.

Reply:
Turns out manuf doesn't work without updating it's database in some cases anyway. So will have to fix that nevertheless.

Reply:
Ok, added a patch, tests working now